doc/release_instructions.md

```diff
@@ -50,7 +58,7 @@ You'll need the following credentials / tools:
 
 6) Run `docker login` (as the release script pushes new versions of the pachd and job-shim binaries to dockerhub)
 
-7) Run `make point-release` or `make VERSION_ADDITIONAL=<rc/version suffix> release-custom`
+7) Run `make point-release` or `make VERSION_ADDITIONAL=RC1 release-candidate`
```

Comment:
I was under the impression we needed to do `rc1` with lowercase to satisfy some Python thing with read the docs or something. Is that no longer the case?